docs/api.rst

```diff
@@ -126,6 +126,7 @@ and all rates are per-generation.
 .. autoclass:: msprime.PopulationParametersChange
 .. autoclass:: msprime.MigrationRateChange
 .. autoclass:: msprime.MassMigration
+.. autoclass:: msprime.SimulationModelChange
```

Comment:
TODO: need to clarify the exact semantics of this WRT to the time argument. Currently it's guaranteed to stop before a given time for DTWF, but for other models it will stop immediately after first event > this. These semantics will be updated at some point to have the more natural <= semantics.

Reply:
can you just change the DTWF behavior to match that of the other models? i.e. the last time it returns is `MAXTIME` (or whatever the variable is called) for that epoch?

Reply:
I've updated this so fix the semantics all round. See #765 if you're interested.